data/assets/scene/solarsystem/planets/earth/noaa-sos/atmosphere/2000_carbon_monoxide.asset

```diff
@@ -0,0 +1,58 @@
+local globeIdentifier = asset.require("../../earth").Earth.Identifier
+
+
+local Name = "Carbon Monoxide - 2000"
+local Identifier = "noaa-sos-atmosphere-carbon_monoxide"
+local Description = [[ Carbon monoxide is known as "The Silent Killer" because it is a colorless, odorless, tasteless gas that is poisonous to humans and other oxygen breathing organisms. It is also naturally occurring in the atmosphere. Only in high concentrations is carbon monoxide deadly. At the Earth's surface the concentration of carbon monoxide is 100 ppb (parts per billion), however, in urban areas it can get up to 10 ppm (parts per million) or 100 times higher. One of the main anthropogenic sources of carbon monoxide in the atmosphere is emissions from automobiles. In areas with heavy traffic the carbon monoxide can be measured at 50 ppm. The natural sources of carbon monoxide include volcanoes and brush burning. Between the anthropogenic and natural sources, scientists estimate that the annual production of carbon monoxide is 2-5 gigatons.  ]]
```

Comment:
Maybe line break the description, so it's readable? 

It's useful to be able to read it when looking at the asset and trying to figure out what the layer contains and whether one wants to add it

Reply:
Same goes for all files ofc

Reply:
Done for all files